pkg/hooks/connection/factory.go

```diff
@@ -98,6 +98,7 @@ func (factory *Factory) GetOrCreate(connectionID structs.ConnID) *Tracker {
 }
 
 func capture(db platform.TestCaseDB, req *http.Request, resp *http.Response, logger *zap.Logger, ctx context.Context, reqTimeTest time.Time, resTimeTest time.Time) {
+	V1Beta1 := models.GetVersion()
```

Comment:
Here, you can use version as the variable name

Reply:
Done